lib/variant-model.js

```diff
@@ -59,6 +59,20 @@ _.extend(Variant.prototype, {
     });
   },
 
+  respondWithVariantFile: function(mockDir, fileName) {
+    var self = this;
+    return self.getJsonResponse(self._id, self._route._path, mockDir, fileName);
+  },
+
+  getJsonResponse: function(variant, path, mockDir, fileName) {
```

Comment:
When I call getJsonResponse from route-model, the _id doesn't seem to be set locally in the variant-model. That is why I pass the variant id.

```createJsonResponse: function(path, dataLocation, fileName) {
    console.log('JSON_RESPONSE');
    return this.respondWith(function(request, reply) {
      var pattern = /{\w+}\/*/g;
      path = path.replace(pattern, '');
      var dataFilePath = dataLocation + path + this._id + '/' + fileName;

      reply.file(dataFilePath);
    });
  },```

Doesn't work, am I doing something wrong? :)